### PR TITLE
Trigger pipelines for azure

### PR DESCRIPTION
**Pull request by tuurek**
## Description
Triggering pipelines for triggering azure build pipelines for main changes and releases.

## Related Issue(s)

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX)**

## Motivation and Context

To allow automated pipeline triggers when code changes.

## How Has This Been Tested?

Tested that pipelines start

### CODE REVIEW

**Review by voneiden — APPROVED** [review]
Looks OK. 

Not an expert on platta pipelines, but could consider if you still want to separate development and production tags to reduce the odds of accidentally deploying a development version into production. In default platta pipelines this can be achieved using `buildenv` parameter, which can have values like `-dev` or `-prod`. Normally a build stores the generated (random) image tag into Azure variable LATEST_IMAGE_TAG. 

Deployment jobs, without a buildenv parameter set, will deploy the image based on what is stored in LATEST_IMAGE_TAG. If buildenv is set for both the build and deployment, for example `buildenv: -dev`, then the build will update both `LATEST_IMAGE_TAG` and `LATEST_IMAGE_TAG_DEV` and the deploy job will use `LATEST_IMAGE_TAG_DEV` when deploying.

File: azure-pipelines-build-release.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # Azure DevOps project/repository
+    name: tiedonohjaus/tiedonohjaus-pipelines
+
+extends:
+  # Filename in Azure DevOps Repository (note possible -ui or -api)
+  # Django example: azure-pipelines-tiedonohjaus-api-release.yml
+  # Drupal example: azure-pipelines-drupal-release.yml
+  template: azure-pipelines-build-tiedonohjaus-api-release-staging.yml@tiedonohjaus-pipelines
+#  parameters:
+    #imagetag: ${{ parameters.imagetag }}
```
Comment (voneiden) [inline azure-pipelines-build-release.yml:47]: Well ok, minor nitpick from unix/linux perspective would be good to have newlines at the end of the yaml files.

**Comment by voneiden** [comment]
By the way, doesn't matter with this one, but apparently sonarflow can't be used on PR's coming from forks, so for future code commits it would be preferable if PR's are made directly from this repo instead of forks.

**Comment by tuurek** [comment]
Added newlines to my fork, pipelines are just a base that can be modified as needed by the project.

**Pull request merged**